### PR TITLE
[Feature] Support coco-stuff dataset

### PR DESCRIPTION
**Pull request by DerrickWang005**
update dataset_prepare.md

### CODE REVIEW

File: mmseg/datasets/coco_stuff.py [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+        [192, 192, 224],
+        [64, 32, 64],
+        [64, 192, 128],
+        [64, 192, 224],
+        [192, 160, 64],
+        [64, 192, 0],
+        [192, 192, 96],
+        [192, 96, 192],
+        [64, 64, 128],
+        [64, 64, 96],
```
Comment (xvjiarui) [inline mmseg/datasets/coco_stuff.py:228]:
```suggestion
        [64, 64, 96]
```

**Comment by xvjiarui** [comment]
Hi @DerrickWang005 

Thanks for the nice PR. We will review and benchmark it soon.

**Comment by DerrickWang005** [comment]
As for the palette, I adopt the PASCAL VOC style. Maybe you can use the official style.😂

**Comment by xvjiarui** [comment]
> As for the palette, I adopt the PASCAL VOC style. Maybe you can use the official style.😂

It would be nice if we could use the official palette.

**Comment by DerrickWang005** [comment]
I found COCO Stuff dataset has two versions, 10k and 164k. After updating that, I will create a new PR ASAP.

**Pull request closed without merging**